server/expression/linting/missing_variables.go

```diff
@@ -0,0 +1,118 @@
+package linting
+
+import (
+	"reflect"
```

Comment:
What does this reflect module does? is it the one that reads the tags you added?

Reply:
It's a module that allows you to get information about a type. You can see its fields, its types, iterate though them, etc.